### PR TITLE
feat(subscriptions): subscriptions CRUD

### PR DESCRIPTION
**Pull request by thewizarodofoz**
## Summary
Create/delete subscriptions in HB

### CODE REVIEW

File: packages/n8n-nodes-honeybook/nodes/HoneyBookTrigger/honeyBookApi.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { IHookFunctions, IHttpRequestMethods } from 'n8n-workflow';
+
+export async function honeyBookApiRequest(
+	this: IHookFunctions,
+	method: IHttpRequestMethods,
+	path: string,
+	body: any = {},
+) {
+	const response = await this.helpers.httpRequestWithAuthentication.call(this, 'honeyBookApi', {
```
Comment (ShragaUser) [inline packages/n8n-nodes-honeybook/nodes/HoneyBookTrigger/honeyBookApi.ts:9]: can we manipulate the credentials here? we should wrap them with a JWT token signed by an ENV var we control, this way we know that each request was made by "N8N" and was not tampered with.

**Review by ShragaUser — APPROVED** [review]
Left a comment, but we can handle that in next PRs

File: packages/n8n-nodes-honeybook/nodes/HoneyBookTrigger/honeyBookApi.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		baseURL: 'http://localhost:8000/api/v2',
```
Comment (ShragaUser) [inline packages/n8n-nodes-honeybook/nodes/HoneyBookTrigger/honeyBookApi.ts:10]: this should accept env var
also - should it include path? as we do send "path"

File: packages/n8n-nodes-honeybook/nodes/HoneyBookTrigger/honeyBookApi.ts (hunk 1 above)
Comment (thewizarodofoz) [reply packages/n8n-nodes-honeybook/nodes/HoneyBookTrigger/honeyBookApi.ts:9]: in a later PR

**Pull request merged**